src/NServiceBus.Core/MessageMutators/Mutators.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NServiceBus.Features
+{
+    class Mutators : Feature
+    {
+        public Mutators()
+        {
+            EnableByDefault();
+        }
+
+        protected internal override void Setup(FeatureConfigurationContext context)
+        {
+            context.Pipeline.Register("MutateIncomingTransportMessage", new MutateIncomingTransportMessageBehavior(), "Executes IMutateIncomingTransportMessages");
```

Comment:
(In general it would definitely make sense to move the mutators into their own feature, but I think that's out of scope here?)